### PR TITLE
fix: Update transaction error modal for a custom message

### PR DESCRIPTION
**Pull request by lolboysg**
This PR aims to fix the issue #205 by adding:
- Custom error messages on transaction modals
- Removing the close icon as it is not possible right now with different bootstrap and react-bootstrap versions
- Capitalize error message
- Creating a custom modal for TransactionError

### CODE REVIEW

**Comment by lolboysg** [comment]
@codynhat @gravenp if you may review

File: components/cards/TransactionError.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+};
+
+function TransactionError(props: TransactionErrorProps) {
+  return (
+    <Alert variant="danger">
+      <Alert.Heading style={{ fontSize: "1em" }}>
+        Transaction Failed
+      </Alert.Heading>
+      <p style={{ fontSize: "0.8em" }}>{`Something went wrong: ${
+        props.message ? props.message : "unknown reason"
```
Comment (gravenp) [inline components/cards/TransactionError.tsx:14]: @lolboysg Can you add a period after the error message?

**Review by gravenp — COMMENTED** [review]
Looks good on the user testing side. 

@codynhat if the refactor approach and code look good, then you can approve & merge.

**Review by codynhat — APPROVED** [review]
Looks good! There are some formatting differences. I will look into setting up prettier in our CI.

**Pull request merged**